xbmc/cores/VideoPlayer/VideoPlayerVideo.cpp

```diff
@@ -697,6 +698,8 @@ bool CVideoPlayerVideo::ProcessDecoderOutput(double &frametime, double &pts)
           break;
         default:
           stereoMode = m_hints.stereo_mode;
+          if (m_processInfo.GetVideoSettings().m_StereoInvert)
```

Comment:
I think this can be performed after `switch` statement after deleting the same from cases.